traversal/common/Identifier.java

```diff
@@ -26,6 +26,7 @@
 
 import static com.vaticle.typedb.common.util.Objects.className;
 import static com.vaticle.typedb.core.common.exception.ErrorMessage.Internal.ILLEGAL_CAST;
+import static com.vaticle.typedb.core.common.exception.ErrorMessage.Internal.ILLEGAL_STATE;
 
 public abstract class Identifier {
```

Comment:
Let's rename `.Name` to `.NamedConcept` and `.Anonymous` to `.AnonymousConcept`. It's not really great but it'll have to get fixed in Rust